app/appsettings.h

```diff
@@ -75,6 +75,9 @@ class AppSettings: public QObject
     const QString &activePositionProviderId() const;
     void setActivePositionProviderId( const QString &id );
 
+    static const QString GROUP_NAME;
```

Comment:
I liked it a bit more before, string defines make me a bit dizzy, but this is matter of preference :)